### DISCUSSION TITLE
Assistance with Run Errors

### DISCUSSION THREAD

**Discussion by youngdie07** [NHERI-SimCenter/SimCenterCommon#230] [Hydro-UQ]
Hi, I'm Youngchul

I’m currently trying to run the manual example using HydroUQ version 4.0 (Validation - Multiple Debris Impacts on a Raised Structure - Digital Twin (OSU LWF)). I followed the manual and entered the EVT values as instructed, but I’ve encountered a few issues:

- The animation on the right side of the GUI, as shown in the manual, is not visible.
- The run failed with the following errors:

 1) Run at DesignSafe: 
`ERROR: NET_REQUEST_PAYLOAD_ERROR Error accessing payload for submitJob request: TAPIS_JSON_VALIDATION_ERROR JSON validation error: TAPIS_JSON_VALIDATION_FAILURE JSON validation error: #/parameterSet/envVariables: array items are not unique #1#/parameterSet/envVariables: array items are not unique`
I’ve been unable to find a solution for these issues. Could you please advise on the next steps to resolve or further investigate this?

![Error_DesignSafe Run](https://github.com/user-attachments/assets/77ad7f64-590b-4084-88c2-771cadb429f9)

 2) Local Run:
`RUNNING DAKOTA ERROR:  b'This program is blocked by group policy. For more information, contact your system administrator.\r` 
and `Error from Dakota: Error: cannot open results file "workdir.1\results.out" for evaluation 1` 
I believe I need to contact my admin regarding this issue, but could you give me some tips on how best to explain the problem to them?

![Error_Local Run](https://github.com/user-attachments/assets/9e75a12e-1c20-4abc-b3f3-e08195ceec1a)

Thank you for your assistance.

**Comment by JustinBonus** [comment]
Apologies for the delayed response,

We are addressing these issues in the upcoming minor release of HydroUQ, though the GUI may remain disabled on some systems. The remotely-ran validation example should soon be working for your system.

Contact bonus@berkeley.edu and fmckenna@berkeley.edu for further support,

Thanks for your continued interest in SimCenter tools,
Justin Bonus
